### PR TITLE
Old potatoes and more condiments.

### PR DESCRIPTION
**Pull request by linuxsudo**
Add a option for old potatoes and added two more condiments.

### CODE REVIEW

File: src/main/java/org/drtshock/Potato.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public static void main(String[] args) {
     }
 
     public void prepare() throws NotDeliciousException {
-        this.addCondiments("sour cream", "chives", "butter", "crumbled bacon", "grated cheese", "ketchup", "salt", "tabasco");
+        this.addCondiments("sour cream", "chives", "butter", "crumbled bacon", "grated cheese", "ketchup", "salt", "tabasco", "pepporoni", "sausage");
```
Comment (drtshock) [inline src/main/java/org/drtshock/Potato.java]: spell check

**Comment by linuxsudo** [comment]
Fixed.

File: src/main/java/org/drtshock/Potato.java [hunk 2]
```diff
@@ -62,6 +77,10 @@ public boolean isDelicious() {
         return this.isBaked();
     }
 
+    public boolean oldPotato() {
```
Comment (drtshock) [inline src/main/java/org/drtshock/Potato.java]: should be isOldPotato

**Comment by linuxsudo** [comment]
It was an extra boolean, fixed.

File: src/main/java/org/drtshock/Potato.java [hunk 3]
```diff
@@ -53,6 +53,21 @@ public boolean isPutIntoOven() {
         }
     }
 
+    public boolean isOldPotato() {
```
Comment (jkcclemens) [inline src/main/java/org/drtshock/Potato.java]: Lots of overhead for checking if this is an old potato. If you're going to return false no matter what, don't query Google! Also, `return false;` in the catch and as the last statement is redundant. You really only need one.

File: src/main/java/org/drtshock/Potato.java [hunk 4]
```diff
@@ -62,6 +77,7 @@ public boolean isDelicious() {
         return this.isBaked();
     }
 
+
```
Comment (jkcclemens) [inline src/main/java/org/drtshock/Potato.java]: :(

**Comment by jkcclemens** [comment]
No activity. Closing.

**Pull request closed without merging**